src/main/java/uk/gov/digital/ho/hocs/casework/client/infoclient/InfoClient.java

```diff
@@ -217,6 +195,13 @@ public UserDto getUser(UUID userUUID) {
         return entityListDtos;
     }
 
+    public EntityDto<Map<String, Object>> getEntityBySimpleName(String simpleName) {
+        EntityDto<Map<String, Object>> entityDto =
```

Comment:
Urgh... is there something else you can use other than `Object`? it's a little too open for abuse in my mind. Don't be afraid to say if not.